backend/impl/build.gradle.kts

```diff
@@ -71,19 +15,23 @@ dependencies {
     testRuntimeOnly(project(":gitCore:jGit"))
 }
 
-//val applySubtypingChecker: Unit by extra
-val shouldRunAllCheckers:Boolean by rootProject.extra
-        if (shouldRunAllCheckers) {
-            dependencies {
-                add("checkerFramework", project(":qual"))
-            }
-            configure<CheckerFrameworkExtension> {
-                checkers.add("org.checkerframework.common.subtyping.SubtypingChecker")
-                val qualClassDir = project(":qual").sourceSets["main"].output.classesDirs.asPath
-                extraJavacArgs.add("-ASubtypingChecker_qualDirs=${qualClassDir}")
-            }
-        }
+BuildUtils.commonsIO(project)
+
```

Comment:
Nit: I'd even remove these separating newlines, just leave newlines as spacing between logical sections (like calls to `BuildUtils` and `CheckerFrameworkConfigurator`)